third_party/move/move-compiler-v2/tests/checking/typing/implicit_deref_borrow_field_non_ref_non_local_root.exp

```diff
@@ -4,7 +4,10 @@ module 0x8675309::M {
         f: u64,
     }
     private fun bar(): M::S {
-        pack M::S(0)
+        {
+          let .f: u64 = 0;
```

Comment:
Can avoid doing the transformation if its not needed? Only the cases were the arguments are out of order need this.